packages/server/src/user/user.resolver.ts

```diff
@@ -38,4 +38,16 @@ export class UserResolver {
       throw new BadRequestException(`Could not find user with ID ${reference.id}`);
     }
   }
+
+  // mutation to update user
+  @Mutation(() => UserModel)
+  @UseGuards(AuthGuard)
```

Comment:
I think in this case you may want to leverage the user decorator to get the authenticated user. With this approach I believe any user could modify any other user's information